migrator/file-helpers.js

```diff
@@ -111,7 +111,7 @@ async function deleteDirectoryIfEmpty (partialPath) {
   }
   const dirContents = await fsp.readdir(dirPath)
   if (dirContents.length === 0) {
-    return await deleteDirectory(dirPath, undefined)
+    return await deleteDirectory(dirPath)
```

Comment:
@BenSurgisonGDS @nataliecarey Looks like this was left behind when the 2nd param was removed